src/cascadia/TerminalApp/AppKeyBindings.cpp

```diff
@@ -143,6 +147,8 @@ namespace winrt::TerminalApp::implementation
     DEFINE_EVENT(AppKeyBindings, DecreaseFontSize,  _DecreaseFontSizeHandlers,  TerminalApp::DecreaseFontSizeEventArgs);
     DEFINE_EVENT(AppKeyBindings, ScrollUp,          _ScrollUpHandlers,          TerminalApp::ScrollUpEventArgs);
     DEFINE_EVENT(AppKeyBindings, ScrollDown,        _ScrollDownHandlers,        TerminalApp::ScrollDownEventArgs);
+    DEFINE_EVENT(AppKeyBindings, ScrollUpPage,	    _ScrollUpPageHandlers,	TerminalApp::ScrollUpPageEventArgs);
```

Comment:
This still isn't lining up according to GitHub. Is that me or is it missing a space still before `_ScrollUpPageHandlers` to align it with the one above it. 

![image](https://user-images.githubusercontent.com/18221333/57657807-63685380-7591-11e9-82bc-3ce852f42540.png)

Reply:
It's not lining up because the _first character_ after `, ScrollUpPage,` is a tab instead of a space. With certain layouts, and certain browsers, it's getting visually compressed to a single space.

Reply:
Probably the same issue below